src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java

```diff
@@ -216,15 +214,15 @@ public void commitPlayerMovement(SearchPath searchPath) {
     }
 
     /**
-      * Integrates the 2 intersections between the bounding circle and the current hint,
-      * then merges the resulting polygon and the remaining possible area
-      * to the new possible area.
+     * Integrates the 2 intersections between the bounding circle and the current hint,
+     * then merges the resulting polygon and the remaining possible area
+     * to the new possible area.
+     * <p>
```

Comment:
If you want to separate two paragraphs i suggest the `<br>` tag

Reply:
also generated by autoformatting